### ISSUE TITLE
Is Dots (Database Opensource Test Suite) part of LTP?

### ISSUE THREAD

**Issue by starquestv** [linux-test-project/ltp#264]
I am trying to locate the latest version of Dots (Database Opensource Test Suite). I see that documentation for Dots 1.1.0 (very old) is posted on the LTP website, but it's unclear to me whether that  is just for convenience or if Dots has been incorporated into LTP.

I did locate Dots 1.1.1, but that is also too old to be of use. In particular, I want to use it with DB2 LUW 10.5, but the DB2 support in Dots 1.1.1 is based on the IBM type 2 JDBC driver and the db2 applet server "db2jstrt", both of which have been deprecated.

If Dots is no longer of use, does anyone have a recommendation for a (relatively) recent JDBC test suite?  Oracle's J2EE API Test Suite 131 looks even older (circa JRE 1.3).

**Comment by pevik** [comment]
@starquestv According to very old LTP page on sourceforge
http://ltp.sourceforge.net/dotshowto.php you can download Dots in sourceforge: https://sourceforge.net/projects/ltp/files/DOTS/.

Dots used to be part of LTP (added in 2006 in 8cc153fa8), but it has been removed long time ago (in 2010 a997c940b).
Were concentrating on testing Linux kernel, not on user space.

**Comment by nqzero** [comment]
@starquestv did you find another database test suite as a replacement ?

**Comment by starquestv** [comment]
Hi nqzero - we ended up using Dots after all.   We are using it with our own JDBC type 4 driver for DB2 (StarSQL for Java), so the lack of support for DB2 LUW 10.5 was not really an issue (though it does give an indication of the age of the test suite)

Dots 1.1.1
downloaded from
https://sourceforge.net/projects/ltp/files/DOTS/Dots-1.1.1/

Note that the test is geared towards performance as much or more than API testing, and a PerfMon server is required. When testing against a database running on another platform (e.g. DB2 for z/OS), I faked things by running the Server portion on a local Linux system with DB2/LUW, even though the performance results are meaningless (we are more interested in the API testing).
